xup/xup.c

```diff
@@ -1358,7 +1358,7 @@ lib_send_client_info(struct mod *mod)
 
     g_writeln("lib_send_client_info:");
     make_stream(s);
-    init_stream(s, 8192);
+    init_stream(s, (sizeof(mod->client_info))+0xFFF & ~0xFFF);
```

Comment:
What is this? I believe such calculations need to have comments. Or they should be done in macros or functions with descriptive names.